### DISCUSSION TITLE
everything

### DISCUSSION THREAD

**Discussion by Basti77** [dani-garcia/vaultwarden#1968] [Ideas]
Hello together.
I would just suggest the option "save everything" in Bitwarden structured.
Again and again you are asked for data or numbers that you guaranteed do not have at hand.
Tax number, health insurance number, social security number...
Birthplace of parents. etc.
Sure, you can store all this in free text, but wouldn't it make sense to introduce country-specific checklists?

Ideally scans of documents.
This is helpful when dealing with authorities and even more so when you have lost all documents in case of a disaster. 
A cell phone is usually with you or an encrypted backup in the Clowd.

**Comment by BlackDex** [comment]
I think you are at the wrong location. This is Vaultwarden, not Bitwarden.
